libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql

```diff
@@ -35,6 +35,12 @@ query GetClientProfile($id: ID!) {
       mailingAddress
       residenceAddress
       importantNotes
+      preferredCommunication
```

Comment:
Hi @Davit-BetterAngels 

I have a few comments.

1. **Primary Phone number icon** 
I didn't provide it for you at the beginning so please use this one when you get a chance.
https://icon-sets.iconify.design/ic/round-star/
The color is: FFC700
2. **Primary Icons** - I noticed even though the 2nd phone number wasn't chosen to be the primary, I still saw the dog paw icon next to it. 
3. **Social Media** - The title should be Social Media without S (sorry my mockup had the s)
4. **Social Media** - When the string gets really long, let's wrap it to the next line. 

<img width="360" alt="image" src="https://github.com/user-attachments/assets/9e70e535-28ae-4cd9-8934-737795651d67">



<img src="https://github.com/user-attachments/assets/fcdeb206-e828-4e50-83a7-e6444ad6414c" width="360" />